proxy/ha_proxy.go

```diff
@@ -227,6 +229,14 @@ func (m HaProxy) getConfigData() ConfigData {
     option  dontlognull
     option  dontlog-normal`
 	}
+
+	default_ports := []string{"80", fmt.Sprintf("443%s", d.CertsString)}
```

Comment:
Hmmm. Not sure I understand the PORT_1. The way I envisioned the cert expansion to work was maybe setting DEFAULT_PORTS="80,8080,443 ssl crt /certs/my-cert.pem,8443 ssl crt /certs/my-cert1.pem". I'm not sure how I understand how the the :ssl expands unless you are implying "ssl" gets replaced by the contents of CertsString. That would make sense too.